content/beginner/150_spotnodegroups/deployapp.md

```diff
@@ -1,30 +1,30 @@
 ---
 title: "Deploy an Application on Spot"
-date: 2018-09-18T17:40:09-05:00
+date: 2021-03-08T10:00:00-08:00
```

Comment:
👍  For the extra yml files in the deployment solutions for all the changes... Good contribution to help customers validate the code and get the final solution if that's what they want !